src/content/home.md

```diff
@@ -1,4 +1,85 @@
 ---
 title: "Home Page"
```

Comment:
The content of this page sounds like a list, I believe will be best implement json or yaml list to generate this page

Reply:
It's not a list, it's the header of the file.
For more information: https://jekyllrb.com/docs/front-matter/

Reply:
Thanks for the improvement, you're right, lists and json better work on this case.